### DISCUSSION TITLE
Tapping ✈️ when title is missing should do something

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#128009] [Mobile] [answered]
GitHub Mobile beta for Android

When writing a long post using the GitHub Mobile application, the title field will have scrolled off screen. 

This leaves the year staring at a disabled ✈️ button. 

![Screenshot_20240610-081636.png](https://github.com/community/community/assets/2119212/b5db999d-0f35-483e-85c2-9c85b181baed)

Tapping the ✈️ button doesn't do anything. 

The user is left to wonder, "what is going wrong?" 

It would be nice if tapping the button did something. 

That could be tossing up a toast "please enter title", or it could be focusing and scrolling to the title field to hint to the user that they must fill it in.

**Comment by elisealix22** [comment, marked as answer]
Hi @jsoref , thank you for reporting this! I agree it would be better to have some feedback here as to why send is disabled and we are now tracking this internally. I appreciate your screenshot and attention to detail in the bug report as well, so thank you.
